code/modules/fish/fish_eggs.dm

```diff
@@ -80,4 +83,18 @@ var/list/fish_eggs_list = list("dud" = /obj/item/fish_eggs,
 	name = "glofish eggs"
 	desc = "A cluster of bright neon eggs belonging to a bio-luminescent species of fish."
 	icon_state = "glofish_eggs"
-	fish_type = "glofish"
\ No newline at end of file
+	fish_type = "glofish"
+
+/obj/item/fish_eggs/seadevil
+	name = "sea devil"
+	desc = "An aquatic offshoot of gremlins that engage in the peculiar behavior of hatching fish eggs to eat mature adults."
+	icon_state = "seadevil"
+	fish_type = "sea devil"
+
+/obj/item/fish_eggs/seadevil/New()
+	..()
+	processing_objects += src
```

Comment:
Note to self, remove from list on Destroy()

Reply:
`obj/Destroy` does it, but doesn't matter that much